roadmap/propagate-native-exception-through-Rust-frame.md

```diff
@@ -40,4 +44,27 @@ on a per-target basis, as the details will vary depending on what
 exception mechanism is in use, and what other non-exception features
 (such as `longjmp`) may use the same mechanism.
 
+[centril-effects]: https://github.com/Centril/rfc-effects/issues/11
+[FAQ-longjmp]: faq.md#how-does-cross-language-unwinding-differ-from-cross-language-setjmplongjmp
+
+## Unwind-termination and related concerns
```

Comment:
These sections in the two files are almost identical. I am wondering whether having two separate files is the right approach.

I'm not yet sure of a better way to organize the information. I suspect a table might be good; I need to think more about what all the columns would be.